vm/src/builtins/object.rs

```diff
@@ -33,6 +34,11 @@ impl PyValue for PyBaseObject {
 
 #[pyimpl(flags(BASETYPE))]
 impl PyBaseObject {
+    #[pyclassmethod(magic)]
+    fn new(cls: PyTypeRef, args: FuncArgs, vm: &VirtualMachine) -> PyResult {
+        let (subtype, args): (PyTypeRef, FuncArgs) = args.bind(vm)?;
```

Comment:
`SlotConstructor` will be preferred for newly added `__new__` with binding.

Reply:
I thought there may be collision with object.tp_new. is it okay to put both?

Reply:
oh! I am sorry. you are right. this is not `tp_new` but just `__new__`.